### PR TITLE
CPed review part2

### PR DESCRIPTION
**Pull request by aap**
like before...

### CODE REVIEW

File: src/peds/Ped.cpp [hunk 1]
```diff
@@ -13909,8 +13898,7 @@ CPed::PedSetQuickDraggedOutCarPositionCB(CAnimBlendAssociation *animAssoc, void
 	if (ped->m_vehEnterType == CAR_DOOR_RF || ped->m_vehEnterType == CAR_DOOR_RR)
 		draggedOutOffset.x = -draggedOutOffset.x;
 
-	finalLocalPos = Multiply3x3(pedMat, draggedOutOffset);
-	finalPos = finalLocalPos + ped->GetPosition();
+	finalPos = Multiply3x3(pedMat, draggedOutOffset) + ped->GetPosition();
```
Comment (erorcun) [inline src/peds/Ped.cpp:13901]: is this original? if so can you show me how can i understand it? except that lgtm

Comment (aap) [reply src/peds/Ped.cpp:13901]: CVector functions use lots of temporaries. a1 below here surely is one so we only have one original variable here
```
      Multiply3x3(&a1, &v18, &a3a);
      finalPos = a1;
      finalPos.x = a1.x + a3->_._._._.m_matrix.m_matrix.pos.x;
      finalPos.y = a1.y + a3->_._._._.m_matrix.m_matrix.pos.y;
      finalPos.z = a1.z + a3->_._._._.m_matrix.m_matrix.pos.z;
```

Comment (erorcun) [reply src/peds/Ped.cpp:13901]: output variables of all Multiply3x3 calls are temporary then?

Comment (aap) [reply src/peds/Ped.cpp:13901]: Hm, probably, yes.

Comment (erorcun) [reply src/peds/Ped.cpp:13901]: ok

**Pull request merged**